static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.js

```diff
@@ -33,11 +34,14 @@ type Links = {
     cookies: string,
 };
 
+const displayEventKey: string = 'first-pv-consent : display';
+const messageCode: string = 'first-pv-consent';
+const blockMessageAfterPageViewNo = 4;
```

Comment:
no type on this...

Reply:
![screen shot 2018-05-30 at 1 58 49 pm](https://user-images.githubusercontent.com/11539094/40722001-09cd6274-6413-11e8-85a4-0b9c6be1eae6.png)